src/pal/inc/pal.h

```diff
@@ -4572,6 +4572,8 @@ YieldProcessor(
     __asm__ __volatile__(
         "rep\n"
         "nop");
+#elif defined(_ARM64_)
+    __asm__ __volatile__( "yield");
```

Comment:
This is the arm64 architectural equivalent of `rep nop`.  In arm64 implementations, it can be ignored, but it is still architecturally correct.